docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -1,41 +1,124 @@
 === macOS Server
 
-This is only required for building iOS applications.
+This is required for building iOS applications. The following specifications are recommended:
+
+* 4GB RAM
+* 2 Cores
+* macOS 10.12 Sierra
+* 30GB minimum disk space and and additional 20GB per XCode version
 
 ==== macOS Machine Setup
 
-- Set `ansible_sudo_pass` variable for the Ansible job to enable the Installer to work with root permissions. It is required by several installation steps.
+Follow the steps below to enable correct configuration of your macOS server via the installation.
```

Comment:
I think we should say here `correct configuration of the jenkins user on your macOS....` as the fulll configuration happens in the ansible playbook.

Reply:
Its more about enabling your machine to be reachable via Ansible. `jenkins` user could be any user we have just used this as default. So yes the full config happens via Ansible, but this doc enables Ansible to do so